test/box-luatest/gh_7255_box_info_before_cfg_test.lua

```diff
@@ -0,0 +1,4 @@
+local t = require('luatest')
+
+local info = box.info()
+t.assert_equals(info.status, "not_configured")
```

Comment:
Shouldn't we add a test function? I guess, in current implementation if it'll fail we'll unable to run any test with `luatest -p <...>`, because the fail will occur on loading test files.

Reply:
Solved.